CHANGELOG.md

```diff
@@ -6,15 +6,29 @@ and follow [semantic versioning](https://semver.org/) for our releases.
 **Breaking Changes** and **Fixed** contain backward incompatible changes, bug fixes, and security patches;
 **Added, Changed, Removed, Deprecated** contain backward compatible improvements or new features.
 
-## [Unreleased](https://github.com/EspressoSystems/jellyfish/compare/0.1.2...main)
+## [Unreleased](https://github.com/EspressoSystems/jellyfish/compare/0.2.0...main)
```

Comment:
Should called v0.2.0 and diff between 0.2.0..0.1.2, as this is the diff this release introduce since last, not against the main.

Then create a new Unreleased with empty Breaking Changes, Updated etc. for post-0.2.0 changes.

Reply:
Previous releases are using tags without prefix "v".
And v0.2.0 CHANGELOG is starting at line 23.